src/test/scala/uk/gov/nationalarchives/tdr/api/routes/ConsignmentRouteSpec.scala

```diff
@@ -643,6 +643,86 @@ class ConsignmentRouteSpec extends TestContainerUtils with Matchers with TestReq
       response.data should equal(expectedResponse.data)
   }
 
+  "consignments" should "allow a user without reporting access to return only his consignments" in withContainers {
```

Comment:
I would reword the test title slightly to: `... to return on their consignments`